doc/raspbian_install.md

```diff
@@ -0,0 +1,130 @@
+
+You'll also hate yourself if, like me, you are dumb enough to do the install on SD media.
+
+An external SSD and an enclosure for it costs less money than you'd spend buying combo meals for four at McDonald's. (Do not try to eat the SSD, no matter how much ketchup you put on it.)
+
+## Dependencies
+
+Here's a partial list of apt dependencies to get things up and running. It doesn't include dependencies that I already had installed thanks to other things running on the . (Ask me how much I want to spin up a completely new Raspbian install to verify this.)
+
+```sh
+sudo apt install gcc-core g++ libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libnss3 libxss1 libasound2 libxtst6 xauth xvfb redis postgresql nginx
+npm i -g pnpm
+```
+
+## Create user and clone repository
+
+We're going to create a special `goblin` user to isolate the goblin install. If you're asked for its home directory, specify it as `/home/goblin`.
+
+We are going to install the app into `/etc/goblin` to isolate it from the rest of the system. (It'll be served over nginx when you do things for real.)
+
+```sh
+sudo adduser goblin
+sudo bash
+cd /etc
+sudo git clone https://github.com/johnHackworth/goblin.git
+sudo chown -R goblin:goblin goblin
+```
+
+## Getting pnpm running
+
+The first thing you'll need to do is get `corepack` up and running so that it can get `pnpm` up and running. We'll install `corepack` underneath `/etc/goblin`.
+
+Note that `pnpm` also needs to do some `npm` stuff, and you'll need to be able to write to a `.npm` directory in $HOME, so let's get that out of the way first:
+
+```sh
+sudo -u goblin bash
+cd /etc/goblin
+corepack use pnpm@8
+```
+
+If you are trying to install to a directory that's on an SD card, go get several cups of coffee and a fun YouTube playlist. You'll be here a while waiting for `corepack use` to finish.
+
+...and then you will find the install fails halfway through! Cypress thinks that you don't satisfy its system requirements. (The Cypress devs do not think people want to use it on a Pi, so their install script doesn't recognize the `armv7` architecture. Their install script defaults to assuming any architecture it doesn't recognize is ia32.) You must fool it. Go into `node_modules/cypress/lib/util.js` and edit the `getRealArch()` function:
+
+```javascript
+async getRealArch() {
+    return 'arm64';
+    // and then the rest of it
+}
+```
+
+Cypress will download a binary that isn't compatible with your Pi, which means you won't be able to run any tests. But you'll at least have succeeded installing pnpm.
+
+## Installing Rust
+
+The `packages/backend/native-utils/` directory contains, well, some native code. It needs `cargo` to be installed. The `cargo` from Raspbian isn't a recent enough version to build with, so you're going to need to install it via the Rust website. We'll install it just for the `goblin` user rather than system-wide for the moment.
```

Comment:
not in scope of this PR but we could provide pre-built binaries for the native-utils? Either on github or on my heptapod, if I figure out if there's any public download feature

On one hand complication could be in building them for all appropriate architectures, but then again it's not like this will be a massive project with thousands of downloads. But at least for low-powered systems like Raspberry it would make sense, since I don't think either of us plans to change the native-utils repo until it breaks with some node.js upgrade.